src/Xamarin.Android.Build.Tasks/Utilities/MonoAndroidHelper.cs

```diff
@@ -278,6 +278,24 @@ public static void LogWarning (object log, string msg, params object [] args)
 		}
 
 #if MSBUILD
+		// Names of assemblies which don't have Mono.Android.dll references, or are framework assemblies, but which
+		// could have Java types.
+		static readonly HashSet<string> SpecialAssemblies = new HashSet<string> (StringComparer.OrdinalIgnoreCase) {
```

Comment:
Why have/use `SpecialAssembles` instead of `KnownAssemblyNames`, in `MonoAndroidHelper.Linker.cs`?